src/main/java/frc/robot/Robot.java

```diff
@@ -17,6 +17,7 @@
 public class Robot extends TimedRobot {
   private Command m_autonomousCommand;
 
+  @SuppressWarnings("unused")
```

Comment:
Do you even need this variable? I see it's assigned on line 31. Will it eventually be used somehow? If not, I would recommend removing it altogether and simply doing `new RobotContainer()` on line 31 with no assignment.

Reply:
Once we start getting fully fleshed subsystems and some commands it will be used, the `@SuppressWarnings` is temporary.